tools/generate_random_db.sh

```diff
@@ -15,17 +15,55 @@ input_data_dir=$1
 db_dir=$2
 rm -rf $db_dir
 
+second_gen_compression_support=
+mixed_compression_support=
+# Support for `ldb --version` is a crude under-approximation for versions
+# supporting dictionary compression and algorithms including zstd and lz4
+if ./ldb --version 2>/dev/null >/dev/null; then
```

Comment:
Why are we suppressing error message as well? Shouldn't we stop proceeding if ./ldb --version shows an error message even when it's excuted?

Reply:
This generate_random_db.sh has to run with many old versions of ldb. This is using whether `ldb --version` succeeds or fails to indicate support for compression-related features that are more difficult to test for. Older buidls will expectedly fail and newer builds will succeed.